staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -82,10 +78,9 @@ type ExplainOptions struct {
 
 func NewExplainOptions(parent string, streams genericclioptions.IOStreams) *ExplainOptions {
 	return &ExplainOptions{
-		IOStreams:       streams,
-		CmdParent:       parent,
-		EnableOpenAPIV3: cmdutil.ExplainOpenapiV3.IsEnabled(),
```

Comment:
Is it time to also remove cmdutil.ExplainOpenapiV3?

Reply:
yes prob